module.txt

```diff
@@ -7,7 +7,7 @@
     "description": "A basic set of world generators and facets.",
     "dependencies": [
         { "id": "BiomesAPI", "minVersion": "4.0.0" },
-        { "id": "CoreAssets", "minVersion": "2.0.1" }
+        { "id": "CoreAssets", "minVersion": "2.2.0-SNAPSHOT" }
```

Comment:
Why "snapshot"?

Reply:
I don't think we should do a module release for each and every little change. We have the `develop` branches for some reason, so maybe we can use that for "staging" multiple changes before doing a full release.

Thus, I think that `-SNAPSHOT` dependencies are okay on `develop` but have be replaced by releases before merging to `master`...

Reply:
While I understand the staging approach, I'm a tad afraid, that this increased maintenance effort will will delay module and as a result also game releases.
But I guess we can do it like that for now and discuss that some place else.